src/entity/Application.ts

```diff
@@ -0,0 +1,60 @@
+import {Column, PrimaryGeneratedColumn, Entity, BeforeInsert, BeforeUpdate, OneToOne} from 'typeorm';
+import { validateOrReject, IsDefined } from 'class-validator';
+
+import {User} from './User';
+
+@Entity()
+export class Application {
+
+    @PrimaryGeneratedColumn()
+    id!: number;
+
+    @Column()
+    completed!: boolean;
+
+    @Column({nullable: true})
+    firstName?: string;
+
+    @Column({nullable: true})
+    lastName?: string;
+
+    @Column({nullable: true})
+    pronouns?: string;
+
+    @Column({nullable: true})
+    email?: string;
+
+    @Column({nullable: true})
+    levelOfStudy?: string;
+
+    @Column({nullable: true})
+    program?: string;
+
+    @Column({nullable: true})
+    shortAnswer1?: string;
+
+    @Column({nullable: true})
+    shortAnswer2?: string;
+
+    @Column({nullable: true})
+    shortAnswer3?: string;
+
+    @Column({nullable: true})
+    numHackathons?: number;
+
+    @Column({nullable: true})
+    siteUrl?: string;
```

Comment:
```suggestion
    personalSiteUrl?: string;
```

Maybe for clarity? Not sure.